index.html

```diff
@@ -38,175 +46,199 @@ <h1>tech<br />Empress</h1>
           </p>
           <a href="#" class="button navy-button">Start Collaboration</a>
         </div>
+        <div class="headline-img">
+          <div class="headline-img1">
+            <img src="./images/image_geometry_2.svg" />
+          </div>
+
+          <div class="headline-img2">
+            <img src="./images/desktop/Thur.svg" />
+          </div>
+          <div class="headline-img3">
+            <img src="./images/desktop/image_dots.svg" />
+          </div>
+        </div>
       </section>
-      <section class="projects">
-        <h2 class="project-heading">Projects</h2>
 
-        <section class="projects">
-          <img
-            src="./images/project1.svg"
-            alt="Project-1"
-            class="project-img"
-          />
-          <div class="project-info">
-            <h3 class="project-name">Project name goes here</h3>
-            <div class="language-info">
-              <span class="language">HTML/CSS</span>
-              <span class="language">Ruby on Rails</span>
-              <span class="language">JavaScript</span>
+      <section class="projects-container">
+        <div class="project-heading">
+          <h2>Projects</h2>
+        </div>
+
+        <div class="project-wrapper">
+          <section class="projects p1">
+            <img
+              src="./images/project1.svg"
+              alt="Project-1"
+              class="project-img"
+            />
+            <div class="project-info pb1">
+              <h3 class="project-name">Project name goes here</h3>
+              <div class="language-info">
+                <span class="language">HTML/CSS</span>
+                <span class="language">Ruby on Rails</span>
+                <span class="language">JavaScript</span>
+              </div>
+              <a href="#" class="button navy-button project-view"
+                >See this project <i class="fa-solid fa-arrow-right"></i
+              ></a>
             </div>
-            <a href="#" class="button navy-button project-view"
-              >See this project <i class="fa-solid fa-arrow-right"></i
-            ></a>
-          </div>
-        </section>
+          </section>
 
-        <section class="projects">
-          <img
-            src="./images/project2.svg"
-            alt="Project-2"
-            class="project-img"
-          />
-          <div class="project-info">
-            <h3 class="project-name">Project name goes here</h3>
-            <div class="language-info">
-              <span class="language">HTML/CSS</span>
-              <span class="language">Ruby on Rails</span>
-              <span class="language">JavaScript</span>
+          <section class="projects p2">
+            <img
+              src="./images/project2.svg"
+              alt="Project-2"
+              class="project-img"
+            />
```

Comment:
Great UI @Suigeneriz 🤝

- [ ] You may please fix the white space below this image. This makes your design different from the Figma template that was provided in the project requirements.
![white space](https://user-images.githubusercontent.com/63917725/178114678-98115813-4039-4374-bf89-8e3c4b083343.PNG)

Reply:
@Suigeneriz great work implementing the changes, kindly fix the white space below the laptop image as previously requested, the gap is still there.

Reply:
@Suigeneriz great work implementing the changes requested.